sample-android-app/src/test/java/com/squareup/sample/authgameapp/TestLauncher.kt

```diff
@@ -0,0 +1,40 @@
+package com.squareup.sample.authgameapp
+
+import com.squareup.workflow.FinishWith
+import com.squareup.workflow.Reaction
+import com.squareup.workflow.Workflow
+import com.squareup.workflow.WorkflowPool
+import com.squareup.workflow.WorkflowPool.Launcher
+import com.squareup.workflow.rx2.EventChannel
+import com.squareup.workflow.rx2.Reactor
+import com.squareup.workflow.rx2.doLaunch
+import io.reactivex.Single
+import io.reactivex.subjects.PublishSubject
+
+/**
+ * Launches pretend [Workflow] instances for tests. Move them through
+ * states by pushing [Reaction]s into [reactions].
+ */
+open class TestLauncher<S : Any, E : Any, O : Any> : Launcher<S, E, O> {
```

Comment:
This seems generally useful, should it be in the library somewhere?

Reply:
Definitely. Let's hide the subject behind a `sendState` method, and then you can drop the rx2 dependencies and put this in the test module.

Reply:
#104